### PR TITLE
Feature/cards layout

### PR DESCRIPTION
**Pull request by weberjavi**
### Description
In this PR I've layed out the main components inside the local scene sidebar (species, protection and partner cards).
Some refactoring on the country data card has also been made (in order to have the loader and back button under the sidebar instead of inside the country data card).
### Designs
[Sidebar designs](https://projects.invisionapp.com/d/main?origin=v7#/console/17484384/421888088/inspect)
### Testing instructions
Select a country on the data globe and check the sidebar
### Feature relevant tickets
Due to the absence of some needed data and to do all charts related tasks on the same PR (the next one I'm gonna open), the tasks have been slightly modified (sorry about that, hopefully the next sprint will have some dedicated team planing time).
So [this task, related to the protection and priority card](https://half-earth-map.atlassian.net/browse/HE-24?atlOrigin=eyJpIjoiOWU1YWZmYjAzMzJmNDUzMzgwMmZiNDFiMjFjYTk2NjAiLCJwIjoiaiJ9) is the one that I started and changed along the way :(
### Comments and clarifications:
@gretacv some bits of data are needed in order to complete the sidebar:
- The four species with smallest range (an endpoint is needed)
- The texts for the metadata modals (to be triggered from the country question mark and from the sources of the data).

### CODE REVIEW

**Comment by kfilip94** [comment]
**THE CURRENT PROTECTION widget**
- Add `:` at the end of the title (should be `THE CURRENT PROTECTION:`)
- Different font styles of the legend items:
image
- Decrease the space between black and gray text on the legend item, on the design it's 5px:
image


**SPECIES COMPOSITION widget**
- Add `:` at the end of the title (should be `SPECIES COMPOSITION:`)
- Change font-size of the legend items text: should be 14px, but it's 16px
image
- Same thing here:
image

File: src/components/about/about-component.jsx [hunk 1]
```diff
@@ -89,17 +91,20 @@ const AboutComponent = ({ setPageTexts, textData, VIEW , openAboutPageAnalyticsE
   return (
     <>
       <button
-        className={styles.aboutButton}
+        className={cx(className, styles.aboutButton)}
```
Comment (kfilip94) [inline src/components/about/about-component.jsx:94]: Hey, shouldn't it be the other way around, to enable overriding all CSS props from `aboutButton` class?
```suggestion
        className={cx(styles.aboutButton, className)}
```

File: src/components/local-scene-card/local-scene-card-styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  flex-direction: column;
+  width: 100%;
+  padding: $sidebar-paddings;
+  > :not(:last-child)::after {
+    content: '';
+    position: relative;
+    bottom: -$sidebar-paddings;
+    display: block;
+    margin: 0 auto;
+    border-bottom: 1px dotted #222;
```
Comment (kfilip94) [inline src/components/local-scene-card/local-scene-card-styles.module.scss:14]:
```suggestion
    border-bottom: 1px dotted $_temporary-dark-text;
```

File: src/components/local-scene-sidebar/local-priority-card/local-priority-card-styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+.wdpaIcon,
+.priorityIcon {
+  min-width: 25px;
+  margin-right: 10px;
+}
+
+.wdpaIcon {
+  fill: #FF6C47;
```
Comment (kfilip94) [inline src/components/local-scene-sidebar/local-priority-card/local-priority-card-styles.module.scss:22]:
```suggestion
  fill: $persimmon;
```

File: src/components/local-scene-sidebar/local-priority-card/local-priority-card-styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+.priorityIcon {
+  fill: #FFD413;
```
Comment (kfilip94) [inline src/components/local-scene-sidebar/local-priority-card/local-priority-card-styles.module.scss:26]: Can we add `#FFD413` color to `settings.scss` file? Or use the similar one that we already have?

**Comment by kfilip94** [comment]
Hey @weberjavi, nice job! I left a few comments above.
Btw. are there any plans for implementing a mobile version of the local scene experience?

**Comment by weberjavi** [comment]
Hey Kamil, thanks for the review. I've updated text and layout styles and comitted your suggestions. Regarding colours I'm gonna stick with the current solution since this are not the definitive ones and will be updated on the next iteration.

**Pull request merged**